src/apolo_app_types/helm/apps/postgres.py

```diff
@@ -220,11 +220,18 @@ async def gen_extra_values(
                 bouncer_repicas=int(input_.pg_bouncer.replicas),
             )
 
+        postgrescluster_crd_name = f"pg-{app_id}"
+        if len(postgrescluster_crd_name) > 37:
```

Comment:
37 MAGIC_NUMBER

Reply:
kind of :D 

left a comment, but let's move it to constant